packages/react-menu/src/components/MenuList/useMenuList.ts

```diff
@@ -68,22 +74,27 @@ export const useMenuList = (
     [findAllFocusable, state.ref],
   );
 
-  const { checkedValues, onCheckedValueChange } = state;
+  const [checkedValues, setCheckedValues] = useControllableValue(state.checkedValues, {});
```

Comment:
```suggestion
  const [checkedValues, setCheckedValues] = useControllableValue(state.checkedValues, state.defaultCheckValues);
```

Actually, this should be something like this as with native primitives you're able to do following:

```tsx
<input />
<input defaultValue="foo" />
<input value="foo" />
```